### PR TITLE
Upgrade ember-cli-babel to remove deprecation warning 

### LINKED ISSUE #7: Deprecation warning in Ember 3.0
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: site-xxx-> ember-light-table -> ember-scrollable -> ember-element-resize-detector -> ember-cli-babel

### PR DESCRIPTION
**Pull request by wojm**
This PR upgrades the version number of ember-babel-cli to remove a deprecation warning. 

Example of deprecation warning:
```
warning ember-addon-genie > ember-cli-babel@5.2.8: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.
warning ember-addon-genie > ember-cli-babel > broccoli-funnel > exists-sync@0.0.4: Please replace with usage of fs.existsSync
warning ember-addon-genie > ember-cli-babel > broccoli-babel-transpiler > babel-core > minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning ember-cli-app-version > ember-cli-babel@5.2.8: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.
warning ember-resolver > ember-cli-babel@5.2.8: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.
warning ember-cli-babel@5.2.8: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.
```

Resolves: https://github.com/EmberSherpa/ember-element-resize-detector/issues/7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,11 +50,11 @@
     "ember-addon"
   ],
   "dependencies": {
+    "broccoli-funnel": "1.0.2",
+    "broccoli-merge-trees": "1.1.1",
     "element-resize-detector": "1.1.4",
     "ember-cli-babel": "^5.1.6",
```
Comment (nicodv) [inline package.json:56]: Shouldn't this be upgraded?

Comment (wojm) [reply package.json:56]: oh yeah. it doesnt actually look like this PR changes anything. I'll check it out

**Comment by allthesignals** [comment]
What's the status on this stuff?

**Comment by Gaurav0** [comment]
Issue fixed.

**Pull request closed without merging**